### PR TITLE
fix: various code issues

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Change descriptions are in git commit message titles.

### CODE REVIEW

File: editable-samples-2/pages/filter/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <link href="../../css/editable.css" rel="stylesheet"></link>
+    <link href="../../css/editable.css" rel="stylesheet" />
     <link href="../../css/prism.css" rel="stylesheet">
```
Comment (teoli2003) [inline editable-samples-2/pages/filter/index.html:7]: We should unify both endings here.
```suggestion
    <link href="../../css/editable.css" rel="stylesheet" />
    <link href="../../css/prism.css" rel="stylesheet" />
```

File: target-text/index.html [hunk 2]
```diff
@@ -78,13 +78,13 @@ <h1>::target-text</h1>
         air? It would require a power equal to 400 horses for the sails of a
         ship to struggle on equal terms with the wind. Suppose an impossibility,
         namely, that a balloon could carry with it a force equal to 400
-        horse-power; this result would be of little use, for under the immense
+        horsepower; this result would be of little use, for under the immense
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply target-text/index.html:81]:
```suggestion
        horse-power; this result would be of little use, for under the immense
```

File: target-text/index.html [hunk 3]
```diff
@@ -78,13 +78,13 @@ <h1>::target-text</h1>
...
         weight the fragile covering of the balloon would instantly collapse. If
         all the horses of a regiment were harnessed to the car of a balloon by
         means of a long rope, the result would be that the balloon would fly
         into shivers, being too fragile to withstand these two opposing forces.
         Man must seek to raise himself in the air by another mode of operation
-        altogether, if he wish to guide himself at the same time. Some time ago
+        altogether, if he wishes to guide himself at the same time. Some time ago
```
Comment (OnkarRuikar) [inline target-text/index.html:87]:
```suggestion
        altogether, if he wish to guide himself at the same time. Some time ago
```

File: target-text/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply target-text/index.html]: reverted both the changes

**Comment by OnkarRuikar** [comment]
I'll run prettier in a separate PR.

**Comment by teoli2003** [comment]
I unfortunately read your comment after merging prettier…

**Pull request merged**